### ISSUE TITLE
Don't use nginx image for the Pod example

### ISSUE THREAD

**Issue by nak3** [openshift/openshift-docs#1533]
https://docs.openshift.com/enterprise/3.1/dev_guide/persistent_volumes.html#claims-as-volumes-in-pods

OpenShift example uses `nginx` image for the pod:

```
      image: "nginx"
```

However, ngingx image needs `super-user privileges` and we can't deploy without [configuration of RunAsAny](https://docs.openshift.org/latest/admin_guide/manage_scc.html#enable-images-to-run-with-user-in-the-dockerfile).

Here is the log outpu of `nginx` image after deploy.

```
$ oc logs -f mypod
nginx: [alert] could not open error log file: open() "/var/log/nginx/error.log" failed (13: Permission denied)
2016/01/08 08:04:11 [warn] 1#1: the "user" directive makes sense only if the master process runs with super-user privileges, ignored in /etc/nginx/nginx.conf:2
2016/01/08 08:04:11 [emerg] 1#1: mkdir() "/var/cache/nginx/client_temp" failed (13: Permission denied)
```

**Comment by nak3** [comment]
Not only [this section](https://docs.openshift.com/enterprise/3.1/dev_guide/persistent_volumes.html#claims-as-volumes-in-pods) but also all of `nginx` image example doesn't work in the doc...

**Comment by chouseknecht** [comment]
+1
Ran into this exact issue while trying to create an OpenShift example for [ansible-container](https://github.com/ansible/ansible-container).

**Comment by mburke5678** [comment]
@nak3 @chouseknecht I did a quick search using the Atom text editor. It reported 88 instances of "nginx" across 13 docs in the OCP docs. Do we need to change all references? Is there another image type that we can find/replace without causing other problems? 
I would appreciate any thoughts. 
cc. @vikram-redhat

**Comment by viper0131** [comment]
What is the current status of this? I ran into the same example errors.

What is the best alternative to use instead of nginx?

**Comment by mburke5678** [comment]
@sjenning Is this something you can help us with?

**Comment by sjenning** [comment]
If I do a search for `image: nginx`, `image: "nginx"`, and `"image": "nginx"`, I get 5 occurrences

How to limit bandwidth
https://github.com/openshift/openshift-docs/blob/master/admin_guide/managing_pods.adoc#L107

How to use PVC as volumes in pods
https://github.com/openshift/openshift-docs/blob/master/dev_guide/persistent_volumes.adoc#L121

How to limit compute resources for a pod
https://github.com/openshift/openshift-docs/blob/master/dev_guide/compute_resources.adoc#L142

Example of an app with a gluster backed PV
https://github.com/openshift/openshift-docs/blob/master/install_config/persistent_storage/topics/glusterfs_using_storage.adoc#L18

Release note with an init container example
https://github.com/openshift/openshift-docs/blob/master/release_notes/ocp_3_6_release_notes.adoc#L221

None of these examples actually _need_ to be nginx to work.   Might use hello-openshift (`openshift/hello-openshift`) instead.

**Comment by mburke5678** [comment]
@sjenning  I got rid of those references. 

Do we want to to anything with these:

**Pods and services** 
```
Consider:
    A pod consisting of an nginx container, with the label role=webserver.
```
https://github.com/openshift/openshift-docs/blob/master/architecture/core_concepts/pods_and_services.adoc

**Quotas and Limit Ranges**
https://github.com/openshift/openshift-docs/blob/master/dev_guide/compute_resources.adoc
Contains numerous nginx references

https://github.com/openshift/openshift-docs/blob/master/install_config/persistent_storage/topics/glusterfs_using_storage.adoc
Contains numerous nginx references

**Mounting Volumes on Privileged Pods**
Contains numerous nginx references
https://github.com/openshift/openshift-docs/blob/master/install_config/storage_examples/privileged_pod_storage.adoc

**Sharing an NFS Persistent Volume**
https://github.com/openshift/openshift-docs/blob/master/install_config/storage_examples/shared_storage.adoc
Contains numerous nginx references

**Comment by sjenning** [comment]
might as well change them all, except this one since it is not part of a spec example
> A pod consisting of an nginx container, with the label role=webserver.

**Comment by mburke5678** [comment]
I manually did a find/replace. Is there someone on your teams who could take a close look to make sure I didn't create any bad references? 
@sjenning PTAL 
@mdshuai PTAL

**Comment by vikram-redhat** [comment]
I am going to close this issue as the original ask has been fixed via #10618 .

**Comment by slmingol** [comment]
I used this container image instead, `bitnami/nginx:latest` as described here - https://medium.com/bitnami-perspectives/non-root-containers-to-show-openshift-some-love-3b32d7218ac6.
